packages/core/admin/admin/src/components/UnstableGuidedTour/Step.tsx

```diff
@@ -111,13 +115,19 @@ const createStepComponents = (tourName: ValidTourName): Step => ({
     </Box>
   ),
 
-  Actions: ({ showStepCount = true, showSkip = false, to, ...props }) => {
+  Actions: ({ showStepCount = true, showSkip = false, to, trackedEvent, ...props }) => {
+    const { trackUsage } = useTracking();
     const dispatch = unstableUseGuidedTour('GuidedTourPopover', (s) => s.dispatch);
     const state = unstableUseGuidedTour('GuidedTourPopover', (s) => s.state);
     const currentStep = state.tours[tourName].currentStep + 1;
     // TODO: Currently all tours do not count their last step, but we should find a way to make this more smart
     const displayedLength = state.tours[tourName].length - 1;
 
+    const handleSkipAction = () => {
+      trackUsage('didSkipGuidedtour');
+      dispatch({ type: 'skip_tour', payload: tourName });
```

Comment:
Is this supposed to be tracking for skipping a single tour or all tours? We should clarify that with product. Maybe we need a new event?

Reply:
Right now a single tour and also the Close guided tour in the homepage